behat/tests/features-v2/reporting/sections/actions/actions.feature

```diff
@@ -0,0 +1,44 @@
+@v2
+Feature: Actions
+
+  Scenario: I complete the actions section with negative responses
+    Given a Lay Deputy logs in and has a new report
+    And I view and start the actions report section
+    Then I should be on the financial decision actions page
+    When I choose no and save on financial decision actions section
+    Then I should be on the concerns actions page
+    When I choose no and save on concerns actions section
+    Then I should be on the actions report last step page
+    And I should see the expected action report section responses
+    When I follow link back to report overview page
+    Then I should be on the Lay reports overview page
+    And I should see "actions" as "finished"
+
+  Scenario: I complete the actions section with positive responses
+    Given a Lay Deputy logs in and has a new report
+    And I view and start the actions report section
+    Then I should be on the financial decision actions page
+    When I choose yes and save on financial decision actions section
+    Then I should be on the concerns actions page
+    When I choose yes and save on concerns actions section
+    Then I should be on the actions report last step page
+    And I should see the expected action report section responses
+    And I should see the expected action comments
+
+  Scenario: I half fill in responses and come back and edit them
+    Given a Lay Deputy logs in and has a new report
+    And I go to the url for the report overview page
+    Then I should see "actions" as "not started"
+    When I view and start the actions report section
+    Then I choose no and save on financial decision actions section
+    And I press report sub section back button
+    And I press report sub section back button
+    Then I should see text asking to answer the question
+    When I follow link back to report overview page
+    Then I should see "actions" as "not finished"
+    When I view the actions report section
+    Then I should be on the actions report summary page
+    When I follow edit link on concerns question
+    And I choose no and save on concerns actions section
+    And I follow link back to report overview page
+    Then I should see "actions" as "finished"
```

Comment:
Nice - I missed this out on the recent scenario I did so will backfill once this is merged.